package.json

```diff
@@ -81,14 +81,13 @@
     "@carbon/charts": "^0.30.18",
     "@carbon/charts-react": "^0.30.18",
     "@carbon/colors": "10.8.0",
-    "@carbon/icons": "10.9.3",
     "@carbon/icons-react": "10.10.0",
     "@carbon/layout": "10.7.1",
     "@carbon/motion": "10.6.0",
     "@carbon/themes": "10.10.3",
     "carbon-components": "10.10.3",
     "carbon-components-react": "7.10.3",
-    "carbon-icons": "7.0.7",
+    "carbon-icons": "^7.0.7",
```

Comment:
Should this be pinned to specific version?

Reply:
I don't think we need it to be pinned anymore since we're not using it. If an application is using a different patch version, npm/yarn should de-dupe it as normal with no effect on our components.

Ideally we'd just match `carbon-components-react` and put it as a peer dep, but consumers are already relying on us for a one-shot install.

Reply:
Are we sure this is still needed? https://github.com/carbon-design-system/carbon/blob/5926a5a9911d6aeab0865e9cf62f466dd696798c/packages/icons-react/package.json#L31

Reply:
Never mind, I misread the above comment.

Reply:
I would say for consistency to pin it but since we hope to refactor away from this eventually not that big of deal.